### ISSUE TITLE
Comfy Quickslots - breaks crafting

### ISSUE THREAD

**Issue by Kherae** [BruceOfTheBow/BruceComfyMods#28]
```
[Error  : Unity Log] MissingFieldException: Field not found: System.Collections.Generic.KeyValuePair`2<Recipe, ItemDrop/ItemData> .InventoryGui.m_selectedRecipe Due to: Could not find field in class
Stack trace:
(wrapper dynamic-method) InventoryGui.DMD<InventoryGui::OnCraftPressed>(InventoryGui)
UnityEngine.Events.InvokableCall.Invoke () (at <be2cce08ca774b9684099a81093ecac0>:0)
UnityEngine.Events.UnityEvent.Invoke () (at <be2cce08ca774b9684099a81093ecac0>:0)
UnityEngine.UI.Button.Press () (at <727fd5405ffb4795abf6c3e1107e7369>:0)
(wrapper dynamic-method) UnityEngine.UI.Button.DMD<UnityEngine.UI.Button::OnPointerClick>(UnityEngine.UI.Button,UnityEngine.EventSystems.PointerEventData)
UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerClickHandler handler, UnityEngine.EventSystems.BaseEventData eventData) (at <727fd5405ffb4795abf6c3e1107e7369>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) (at <727fd5405ffb4795abf6c3e1107e7369>:0)
UnityEngine.EventSystems.EventSystem:Update()
```

can no longer craft with comfy quickslots enabled, as of the bog witch update.

**Comment by Elrondhoover** [comment]
I am also having this problem where I cannot craft with Comfy Quickslots enabled, it also interferes with devcommands. Disabling the mod cures both problems.

**Comment by redseiko** [comment]
Fixed with version `v1.7.0`.
* https://thunderstore.io/c/valheim/p/ComfyMods/ComfyQuickSlots/
